packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -156,6 +156,51 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
         ></p>
     </div>
 
+    <div
+        class="widget-presentation"
+        data-ng-if="!$ctrl.loading.migrate && $ctrl.process.selectedOffer.totalSubServiceToDelete > 0"
+    >
+        <h2
+            class="widget-presentation-title"
+            data-translate="telecom_pack_migration_confirm_resume_contact_owner"
+        ></h2>
+        <div class="d-inline-flex p-2 bd-highlight">
+            <div
+                class="form-group"
+                data-ng-class="{
+                    'has-error': resumeForm.contactPhone.$touched && resumeForm.contactPhone.$invalid
+                }"
+            >
+                <label
+                    for="contactPhone"
+                    class="control-label"
+                    data-translate="telecom_pack_migration_confirm_resume_contact_phone"
+                >
+                </label>
+                <input
+                    type="text"
+                    class="form-control"
+                    id="contactPhone"
+                    name="contactPhone"
+                    data-ui-validate="{ pattern: '$ctrl.checkContactPhone($value)' }"
+                    data-ng-model="$ctrl.contactPhone"
```

Comment:
Why not directly
```suggestion
                    data-ng-model="$ctrl.process.contactPhone"
```
?